Cargo.toml

```diff
@@ -1,23 +1,34 @@
 [package]
 name = "frl-proxy"
-version = "0.5.1"
-authors = ["Andrew Dorton <adorton@adobe.com>"]
+version = "0.9.0"
+authors = ["Andrew Dorton <adorton@adobe.com>", "Daniel Brotsky <dbrotsky@adobe.com>"]
 edition = "2018"
 
 [dependencies]
-hyper = "0.13"
-hyper-tls = "0.4"
-tokio = { version = "0.2", features = ["full"] }
-tokio-rustls = "^0.14.0"
+tokio = { version = "1.0.1", features = ["full"] }
+hyper = { version = "0.14.2", features = ["full"] }
+hyper-tls = "0.5"
+native-tls = "0.2.7"
+tokio-native-tls = "0.3"
+async-stream = "0.3.0"
+futures = "0.3.1"
+futures-util = "0.3.1"
 config = "0.10"
-serde = { version = "^1.0.8", features = ["derive"] }
+serde = { version = "1.0.118", features = ["derive"] }
+serde_json = "1.0.61"
 structopt = "0.3"
 eyre = "0.6"
-rustls = "0.18"
-futures = "0.3"
-futures-util = "^0.3.1"
 log = "0.4"
 fern = "0.6"
 chrono = "0.4"
 ctrlc = "3.1"
 openssl-probe = "0.1.2"
+#sqlx = { version = "0.4.2", default-features = false, features = [ "runtime-tokio-native-tls", "sqlite" ] }
+#sqlx = { path="../sqlx", features = [ "runtime-tokio-native-tls", "sqlite" ] }
+url = "2.1.1"
+sys-info = "0.7.0"
+
+[dependencies.sqlx]
+git = "https://github.com/ant32/sqlx"
```

Comment:
FWIW, I usually delete branches after they've been merged. I would not consider anything in the fork stable.

Reply:
Gotcha, thank you.

@adobeDan - I've forked the repo I suggested above - let's use `tokio-1.0` on that until we get the update in a release.

```
https://github.com/adorton-adobe/sqlx
```

Reply:
I completely agree that we need our own pet repo holding the hacked sqlx, and I will use yours for now.  It looks like the sqlx folks are actually going to release a beta that has this code in it; that would be more reliable and allow us to use a standard release tag instead of our own repo.